.github/workflows/rust.yml

```diff
@@ -66,6 +66,26 @@ jobs:
         with:
           command: build
           args: --target wasm32-unknown-unknown --package minimint-mint
+      - name: Build modules/minimint-ln
+        uses: actions-rs/cargo@v1
+        with:
+          command: build
+          args: --target wasm32-unknown-unknown --package minimint-ln
+      - name: Build modules/minimint-wallet
```

Comment:
Is there a reason to build modules separately and not just the client library?

Reply:
I just forgot to remove them at end.